src/main/java/org/candlepin/pki/certs/SCACertificateGenerator.java

```diff
@@ -345,16 +358,15 @@ private Set<X509Extension> prepareV3ByteExtensions(org.candlepin.model.dto.Produ
     }
 
     private byte[] createContentAccessDataPayload(Consumer consumer,
-        Map<org.candlepin.model.Content, Boolean> ownerContent, PromotedContent promotedContent) {
+        Map<String, ProductContent> activateContent, PromotedContent promotedContent) {
 
         String consumerUuid = consumer != null ? consumer.getUuid() : null;
         log.info("Generating SCA payload for consumer \"{}\"...", consumerUuid);
 
         Product engProduct = new Product()
             .setId("content_access")
-            .setName(" Content Access");
-
-        ownerContent.forEach(engProduct::addContent);
+            .setName(" Content Access")
```

Comment:
Is this sapece also accident? @nikosmoum

Reply:
Yes, but we cannot change this now because SCA certs have been like this for years, and people out there may be parsing them and expecting this space to be there. The anonymous certs aren't 'released' anywhere yet, that's why we can change them

Reply:
ack, thanks